pages/osx/pmset.md

```diff
@@ -0,0 +1,28 @@
+# pmset
+
+> Configure macOS power management settings, as one might do in System Preferences > Energy Saver.
+> These commands must be run as root, i.e. starting with `sudo`.
+
+- See the current power management settings:
+
+`pmset -g`
+
+- See current power source and battery levels:
+
+`pmset -g batt`
+
+- Set display to never sleep when on charger power:
+
+`pmset -c displaysleep 0`
+
+- Have display sleep after 15 minutes when on battery power:
+
+`pmset -b displaysleep 15`
+
+- Restore to system defaults:
```

Comment:
Thanks, you need not had done the force update btw :). Incremental commits are totally fine and help understand the changes done.